vcd/config_test.go

```diff
@@ -43,8 +43,14 @@ type TestConfig struct {
 		MaxRetryTimeout          int    `json:"maxRetryTimeout"`
 	} `json:"provider"`
 	VCD struct {
-		Org     string `json:"org"`
-		Vdc     string `json:"vdc"`
+		Org             string `json:"org"`
+		Vdc             string `json:"vdc"`
+		AllocationModel string `json:"allocationModel"`
```

Comment:
IMHO we don't need to hardcode one `AllocationModel` setting in the config. Acceptance tests can just include multiple options to test as I wouldn't want to change config manually for running test for each case.

Reply:
allocation model removed, test improved to cover all cases